explainer.md

```diff
@@ -94,12 +95,14 @@ const host = 'example.com';
 const port = 10001;
 const transport = new QuicTransport(host, port);
 
-setInterval(() => {
+setInterval(async () => {
   // App-specific encoded game state
   const gameState = getGameState();
   const encodeGameState = encodeGameState(gameState);
   const stream = await transport.createSendStream();
-  stream.write({data: encodedGameState, finished: true});
+  const writer = stream.writable.getWriter();
+  writer.write(encodedGameState);
```

Comment:
Do you want to do something similar here for waiting for ready?

Reply:
I'm not sure if it's needed, but I'll add it for now.